### PR TITLE
Bugfix SLACK api validation

### PR DESCRIPTION
**Pull request by babenek**
Uncaught exception might be happen in case wrong server response.

### CODE REVIEW

**Review by csh519 — COMMENTED** [review]
Could you explain more about error?
If there are exception cases that we didn't covered, we should add them, not just make them pass.

File: credsweeper/validations/slack_token_validation.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import logging
```
Comment (csh519) [inline credsweeper/validations/slack_token_validation.py:1]: Flake8 detected `import logging` is unused.
Please check.

Comment (csh519) [reply credsweeper/validations/slack_token_validation.py]: Thank you for explanation.

**Review by csh519 — APPROVED** [review]
Maybe `except Exception:` can be replace with `finally:`. But it is okay.

LGTM :+1:

**Review by kmnls — APPROVED** [review]

**Pull request merged**